bittensor/_metagraph/__init__.py

```diff
@@ -113,8 +113,9 @@ def __init__(self, netuid: int, network: str = 'finney', lite:bool = True, sync:
         if sync:
             self.sync( block = None, lite = lite )
 
-    def sync ( self, block: Optional[int] = None, lite: bool = True ) -> 'metagraph':
-        subtensor = bittensor.subtensor( network = self.network )
+    def sync ( self, block: Optional[int] = None, lite: bool = True, subtensor: Optional['bittensor.Subtensor'] = None ) -> 'metagraph':
+        if not subtensor:
+            subtensor = bittensor.subtensor( config = self.config, network = self.network )
```

Comment:
metagraph object do not have config anymore

Reply:
my bad, removed